models/inspire.py

```diff
@@ -0,0 +1,44 @@
+from bs4 import BeautifulSoup
+import pandas as pd
+import requests
+import urllib.request
+import time
+import random
+import json
+from models import base_db_model
+
+
+class Inspire(base_db_model.BaseDBModel):
+    def __init__(self):
+        pass
+    def get_cs_satire(self):
+        url = 'http://quotes.stormconsultancy.co.uk/random.json'
+        r = requests.get(url)
+        quote = r.json()
+        return quote['author'] + ':' + quote['quote']
+
+    def get_inspiration(self):
+        url = 'https://zenquotes.io/api/random'
+        r = requests.get(url)
+        quote = r.json()
+        return quote[0]['a'] + ': ' + quote[0]['q']
+
+    def get_cs_motivated(self):
+        rand_num = random.randint(1, 189)
```

Comment:
should this not come after the quotes and authors are pulled in? that way you can do length of array incase api changes.